Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx

```diff
@@ -91,9 +95,13 @@ void Tracker::clustersToTracks(std::function<void(std::string s)> logger, std::f
           &Tracker::computeCells, "Cell finding", [](std::string) {}, iteration);
         nCells += mTraits->getTFNumberOfCells();
         if (!mTimeFrame->checkMemory(mTrkParams[iteration].MaxMemory)) {
-          mTimeFrame->printROFInfo(iROFs);
+          mTimeFrame->printSliceInfo(iROFs, mTrkParams[iteration].nROFsPerIterations);
           error(fmt::format("Too much memory used during cell finding in iteration {} in ROF span {}-{}: {:.2f} GB. Current limit is {:.2f} GB, check the detector status and/or the selections.",
                             iteration, iROFs, iROFs + mTrkParams[iteration].nROFsPerIterations, mTimeFrame->getArtefactsMemory() / GB, mTrkParams[iteration].MaxMemory / GB));
+          if (mTrkParams[iteration].DropTFUponFailure) {
+            dropTF = true;
+          }
+          dropTF = true;
```

Comment:
here you set `dropTF = true` once following the condition then unconditionally. I guess the second one is not needed?

Reply:
yep, this is a mistake